data/coalition/coalition ships.txt

```diff
@@ -382,6 +660,183 @@ ship "Heliarch Interdictor" "Heliarch Interdictor (Missile)"
 		"Hyperdrive"
 
 
+ship "Heliarch Judicator"
+	sprite "ship/heliarch judicator"
+	thumbnail "thumbnail/heliarch judicator"
+	attributes
+		category "Medium Warship"
+		licenses
+			Heliarch
+		"cost" 33953000
+		"shields" 47700
+		"hull" 41900
+		"required crew" 31
+		"bunks" 110
+		"mass" 570
+		"drag" 7.9
+		"heat dissipation" .67
+		"ion protection" .2
+		"fuel capacity" 900
+		"cargo space" 136
+		"outfit space" 524
+		"weapon capacity" 164
+		"engine capacity" 114
+		weapon
+			"blast radius" 250
+			"shield damage" 2800
+			"hull damage" 1400
+			"hit force" 3800
+	outfits
+		"Finisher Pod" 2
+		"Finisher Torpedo" 120
+		"Finisher Storage Tube" 2
+		"Ion Hail Turret"
+		
+		"Large Reactor Module"
+		"Small Battery Module"
+		"Large Battery Module"
+		"Overcharged Shield Module"
+		"Overclocked Repair Module" 2
+		"Cooling Module" 2
+		"Scanning Module" 4
+		"Enforcer Riot Staff" 83
+		"Enforcer Confrontation Gear" 49
+		
+		"Small Thrust Module" 2
+		"Small Steering Module"
+		"Large Thrust Module" 2
+		"Large Steering Module"
+		"Scram Drive"
+	
+	engine -33 108 .5
+	engine -17 163 .7
+	engine 17 163 .7
+	engine 33 108 .5
+	gun -13 -124.5 "Finisher Pod"
+	gun 13 -124.5 "Finisher Pod"
+	turret 0 45 "Ion Hail Turret"
+	fighter -34.5 -82
+	fighter 34.5 -82
+	fighter -67.5 35
+	fighter -67.5 35
+	fighter 67.5 35
+	fighter 67.5 35
+	explode "small explosion" 20
+	explode "medium explosion" 50
+	explode "large explosion" 20
+	explode "huge explosion" 10
+	"final explode" "final explosion large" 1
+	description `During their time battling the Quarg, the species of the Coalition focused on producing the strongest, toughest warships they could, so fightercraft was something out of question. Now however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently started experimenting with such ships, and out of a need for something to carry them, the Judicator was made.`
```

Comment:
```suggestion
	description `During their time battling the Quarg, the species of the Coalition focused on producing the strongest, toughest warships they could, so fightercraft were out of question. Now, however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently started experimenting with such ships. Out of a need for something to carry them, the Judicator was made.`
```
OR
```suggestion
	description `During their time battling the Quarg, the species of the Coalition focused on producing the strongest, toughest warships they could, so fightercraft were out of question. Now, however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently experimented with such ships. The Judicator was made for transportation of these fightercraft.`
```
First is purely corrections, second is suggested stylistic changes (optional).